tests/presentations/presentation-instance-dependencies.php

```diff
@@ -80,17 +80,17 @@ trait Presentation_Instance_Dependencies {
 	 * @param Indexable_Presentation $presentation_instance The indexable presentation instance.
 	 */
 	protected function set_instance_dependencies( Indexable_Presentation $presentation_instance ) {
-		$this->options_helper      = Mockery::mock( Options_Helper::class );
-		$this->image_helper        = Mockery::mock( Image_Helper::class );
+		$this->options      = Mockery::mock( Options_Helper::class );
+		$this->image        = Mockery::mock( Image_Helper::class );
 		$this->current_page_helper = Mockery::mock( Current_Page_Helper::class );
```

Comment:
It there a reason that you didn't removed `_helper` from this line and the lines below (86-89)